src/components/BigNumberPad.js

```diff
@@ -0,0 +1,72 @@
+import React, {PureComponent} from 'react';
+import {
+    Text, TouchableOpacity, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+
+const propTypes = {
+    numberPressed: PropTypes.func.isRequired,
+};
+
+const padNumbers = [
+    ['1', '2', '3'],
+    ['4', '5', '6'],
+    ['7', '8', '9'],
+    ['.', '0', '<'],
+];
+
+class BigNumberPad extends PureComponent {
+    /**
+     * Creates set of buttons for given row
+     *
+     * @param {number} row
+     * @returns {View}
+     */
+    createNumberPadRow(row) {
+        const self = this;
+        const numberPadRow = padNumbers[row].map((_column, index) => self.createNumberPadButton(row, index));
+        return (
+            <View key={row} style={[styles.flexRow, styles.mt3]}>
+                {numberPadRow}
+            </View>
+        );
+    }
+
+    /**
+     * Creates a button for given row and column
+     *
+     * @param {number} row
+     * @param {number} column
+     * @returns {View}
+     */
+    createNumberPadButton(row, column) {
+        const marginRight = column < 2 ? styles.mr3 : {};
```

Comment:
`column < 2` and especially `styles.mr3` is a bit obscure for people without knowledge of the whole codebase. What is this line doing to the first 2 columns exactly? Can we perhaps add a comment explaining that?